### PR TITLE
fix rapidjson/cci.20200410 checksum

### LINKED ISSUE #9212: [package] rapidjson/cci.20200410: zip checksum has changed
### Package and Environment Details
  * Package Name/Version: **rapidjson/cci.20200410:**
  * Operating System+version: **Arch Linux**
  * Compiler+version: **GCC 11.1**
  * Conan version: **conan 1.43.2**
  * Python version: **Python 3.10.1**

### Steps to reproduce (Include if Applicable)
Build any package that depends on `rapidjson/cci.20200410`

### Logs (Include/Attach if Applicable)
<details><summary>Click to expand log</summary>

```
ERROR: rapidjson/cci.20200410: Error in source() method, line 21
	tools.get(**self.conan_data["sources"][self.version])
	ConanException: sha256 signature failed for '8f4c021fa2f1e001d2376095928fc0532adf2ae6.zip' file. 
 Provided signature: e6fc99c7df7f29995838a764dd68df87b71db360f7727ace467b21b82c85efda  
 Computed signature: 90b511838618f1f92a0dd9a0ac23a1de436ffaa4dc891b0e5bda66a5469587cc
```

</details>

### PR DESCRIPTION
**Pull request by fardragon**
Specify library name and version:  **rapidjson/cci.20200410**

fixes #9212

### CODE REVIEW

File: recipes/rapidjson/all/conandata.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ sources:
     sha256: bf7ced29704a1e696fbccf2a2b4ea068e7774fa37f6d7dd4039d0787f8bed98e
   "cci.20200410":
     url: "https://github.com/Tencent/rapidjson/archive/8f4c021fa2f1e001d2376095928fc0532adf2ae6.zip"
-    sha256: e6fc99c7df7f29995838a764dd68df87b71db360f7727ace467b21b82c85efda
+    sha256: 90b511838618f1f92a0dd9a0ac23a1de436ffaa4dc891b0e5bda66a5469587cc
```
Comment (Croydon) [inline recipes/rapidjson/all/conandata.yml:7]: Why did it change?

**Review by samuel-emrys — COMMENTED** [review]
```
$ wget https://github.com/Tencent/rapidjson/archive/8f4c021fa2f1e001d2376095928fc0532adf2ae6.zip
--2022-02-02 00:01:01--  https://github.com/Tencent/rapidjson/archive/8f4c021fa2f1e001d2376095928fc0532adf2ae6.zip
SSL_INIT                                                                                                                                   Loaded CA certificate '/etc/ssl/certs/ca-certificates.crt'
Resolving github.com (github.com)... 52.64.108.95                                                                                          
Connecting to github.com (github.com)|52.64.108.95|:443... connected.                                                                      
HTTP request sent, awaiting response... 302 Found                                                                                          
Location: https://codeload.github.com/Tencent/rapidjson/zip/8f4c021fa2f1e001d2376095928fc0532adf2ae6 [following]                 
--2022-02-02 00:01:01--  https://codeload.github.com/Tencent/rapidjson/zip/8f4c021fa2f1e001d2376095928fc0532adf2ae6
SSL_INIT                                                                                                                                   
Resolving codeload.github.com (codeload.github.com)... 54.79.46.55                                                                         
Connecting to codeload.github.com (codeload.github.com)|54.79.46.55|:443... connected.                    
HTTP request sent, awaiting response... 200 OK                     
Length: unspecified [application/zip]                                                                                                      
Saving to: '8f4c021fa2f1e001d2376095928fc0532adf2ae6.zip'                                                                                  
                                                                                                                                           
8f4c021fa2f1e001d2376095928fc0532a     [ <=>                                                            ]   1.16M  6.76MB/s    in 0.2s    
                                                                                                                                           2022-02-02 00:01:02 (6.76 MB/s) - '8f4c021fa2f1e001d2376095928fc0532adf2ae6.zip' saved [1218995]
$ sha256sum 8f4c021fa2f1e001d2376095928fc0532adf2ae6.zip
e6fc99c7df7f29995838a764dd68df87b71db360f7727ace467b21b82c85efda  8f4c021fa2f1e001d2376095928fc0532adf2ae6.zip
```
For the sake of being explicit, as can be seen above, the generated hash is
```
e6fc99c7df7f29995838a764dd68df87b71db360f7727ace467b21b82c85efda
```
Which is the existing hash - it doesn't look to me like this needs to be changed

**Comment by fardragon** [comment]
```                                                                                                                                                                                                                    ➜ wget https://github.com/Tencent/rapidjson/archive/8f4c021fa2f1e001d2376095928fc0532adf2ae6.zip                                                                                                                                                             
--2022-02-01 11:43:11--  https://github.com/Tencent/rapidjson/archive/8f4c021fa2f1e001d2376095928fc0532adf2ae6.zip                                                                                                                                           
SSL_INIT                                                                                                                                                                                                                                                     
Loaded CA certificate '/etc/ssl/certs/ca-certificates.crt'                                                                                                                                                                                                   
Resolving github.com (github.com)... 140.82.121.4                                                                                                                                                                                                            
Connecting to github.com (github.com)|140.82.121.4|:443... connected.                                                                                                                                                                                        
HTTP request sent, awaiting response... 302 Found                                                                                                                                                                                                            
Location: https://codeload.github.com/Tencent/rapidjson/zip/8f4c021fa2f1e001d2376095928fc0532adf2ae6 [following]                                                                                                                                             
--2022-02-01 11:43:11--  https://codeload.github.com/Tencent/rapidjson/zip/8f4c021fa2f1e001d2376095928fc0532adf2ae6                                                                                                                                          
SSL_INIT                                                                                                                                                                                                                                                     
Resolving codeload.github.com (codeload.github.com)... 140.82.121.9                                                                                                                                                                                          
Connecting to codeload.github.com (codeload.github.com)|140.82.121.9|:443... connected.                                                                                                                                                                      
sHTTP request sent, awaiting response... 200 OK                                                                                                                                                                                                              
Length: 1218995 (1.2M) [application/zip]                                                                                                                                                                                                                     
Saving to: ‘8f4c021fa2f1e001d2376095928fc0532adf2ae6.zip’                                                                                                                                                                                                    
                                                                                                                                                                                                                                                             
8f4c021fa2f1e001d2376095928fc0532adf2ae6. 100%[=====================================================================================>]   1.16M  7.40MB/s    in 0.2s                                                                                          
                                                                                                                                                                                                                                                             
2022-02-01 11:43:12 (7.40 MB/s) - ‘8f4c021fa2f1e001d2376095928fc0532adf2ae6.zip’ saved [1218995/1218995]                                                                                                                                                     
                                                                                                                              
                         
➜ sha256sum 8f4c021fa2f1e001d2376095928fc0532adf2ae6.zip                                                                      
90b511838618f1f92a0dd9a0ac23a1de436ffaa4dc891b0e5bda66a5469587cc  8f4c021fa2f1e001d2376095928fc0532adf2ae6.zip 
```

File: recipes/rapidjson/all/conandata.yml (hunk 1 above)
Comment (fardragon) [reply recipes/rapidjson/all/conandata.yml:7]: No clue, something on github side, but it seems it's back to the old hash now

**Comment by fardragon** [comment]
I'll close this PR for now, as it seems that github reverted whatever changes they made that caused the checksums to change

**Pull request closed without merging**